docs/design.md

```diff
@@ -166,7 +174,7 @@ drawbacks, for example:
 To resolve the issues above, you can run OpenTelemetry Collector as an Agent.
 The Agent runs as a daemon in the VM/container and can be deployed independent
 of Library. Once Agent is deployed and running, it should be able to retrieve
-spans/stats/metrics from Library, export them to other backends. We MAY also
```

Comment:
s/spans/traces?

Reply:
I think "spans" is correct.  The data the collector operates on are individual spans.  The data model doesn't even have a "trace" entity, just a collection of "spans".

Reply:
Right, but the configuration for pipelines references `traces`, which aligns with the signal terminology for spans. I guess this is a bit of a grey area :(

Reply:
The reason to use "traces" is because that's how we name the pipeline type.